libs/designsystem/header/src/header.component.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { createHostFactory, Spectator } from '@ngneat/spectator';
+
+import { TestHelper } from '@kirbydesign/designsystem/testing';
+
+import { FlagComponent } from 'flag/src';
+import { AvatarComponent } from 'avatar/src';
+import { HeaderModule } from './header.module';
+import { HeaderComponent } from './header.component';
+
+const avatarTagName = 'kirby-avatar';
+const getAvatarElement = (spectator: Spectator<HeaderComponent>) => spectator.query(avatarTagName);
+const flagTagName = 'kirby-flag';
+const getFlagElement = (spectator: Spectator<HeaderComponent>) => spectator.query(flagTagName);
+const titleSelector = 'h1.title';
+const getTitleElement = (spectator: Spectator<HeaderComponent>) => spectator.query(titleSelector);
+const keySelector = 'h1.key';
+const getKeyElement = (spectator: Spectator<HeaderComponent>) => spectator.query(keySelector);
+const valueSelector = 'h3.value';
+const getValueElement = (spectator: Spectator<HeaderComponent>) => spectator.query(valueSelector);
+const valueUnitSelector = '.value-unit';
+const getValueUnitElement = (spectator: Spectator<HeaderComponent>) =>
+  spectator.query(valueUnitSelector);
+const subtitle1Selector = '.subtitle1';
+const getSubtitle1Element = (spectator: Spectator<HeaderComponent>) =>
+  spectator.query(subtitle1Selector);
+const subtitle2Selector = '.subtitle2';
+const getSubtitle2Element = (spectator: Spectator<HeaderComponent>) =>
+  spectator.query(subtitle2Selector);
+
+describe('HeaderComponent', () => {
+  let createHost;
+  createHost = createHostFactory({
+    component: HeaderComponent,
+    imports: [TestHelper.ionicModuleForTest, HeaderModule, FlagComponent, AvatarComponent],
+    detectChanges: true,
+  });
+
+  describe('Default', () => {
```

Comment:
Consider aligning with rest of test code:
```suggestion
  describe('by default', () => {
```